### ISSUE TITLE
Converting to locale date subtracts 7 minutes

### ISSUE THREAD

**Issue by gabriel-ayusso** [moment/moment#5988]
When I try to convert a date in the past (noticed that it only occurs before 1900's) to locale time from utc, moment is subtracting 7 minutes from the final result. 

```js
moment.utc('1900-01-01T14:00:00Z').local().format('DD/MM/yyyy HH:mm');
// shows => '01/01/1900 10:53'
```

Thanks a lot.

**Comment by sciffany** [comment]
I'm new here and I've been trying to search for leap second calculations in the repo in order to explain this result and learn more about time. But in the end, perhaps it's a matter of historical convention for different time zones? https://stackoverflow.com/questions/66680634/what-is-going-on-with-this-historical-timezone-offset-utc0057

**Comment by mattjohnsonpint** [comment]
Yes.  Chances are this is related to LMT offsets in the tzdata.  It would be hard to reproduce without knowing the exact environment, OS,  browser/node version, tzdata version, etc.  But generally speaking, LMT (local mean time) adjustments are the first entry in the tzdata file for a time zone that moved from some local reference to align with UTC.  This is not a moment-specific phenomenon.
